py3status/modules/process_status.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+Display if a file or dir exists.
+
+Configuration parameters:
```

Comment:
please also state the defaults (and I think that process should have no default)